packages/cli/src/templates/project/package-json.ts

```diff
@@ -31,6 +33,7 @@ export const template = `{
   "repository": "{{{repository}}}",
   "scripts": {
     "lint": "eslint --ext '.js,.ts' **/*.ts",
+    "fix-lint": "eslint --quiet --fix --ext '.js,.ts' **/*.ts",
```

Comment:
great stuff, what do you think about renaming these scripts to `lint:check` and `lint:fix`? those changes may also require refactoring some pipeline steps

Reply:
@tainguyenbui I can do that if you think those are better names or at least more common ones (I think so). However, take into account that this is the template to generate a project, it's not the Booster project itself, so what do you mean with `may also require refactoring some pipelines steps`? Do you mean to make the same change in the Booster project? Let me know, and I'll rename those actions only here or on both sides.

Reply:
@glammers1, in that case, it shouldn't break anything. Yes indeed, the use of `:` is a more standard way, and also makes it tidier

Reply:
Done f1a7c7fac56ed6844f76ec602c2732498ea28739. I also opened an issue #96 to make the same change in Booster.